src/test/java/com/wikia/webdriver/testcases/mercurytests/PortableInfoboxTests.java

```diff
@@ -132,4 +133,40 @@ public void MercuryPortableInfoboxTest_006_DifferentClickTargets() {
         .clickGalleryButton(0)
         .isInfoboxExpanded();
   }
+
+  @Test(groups = "MercuryPortableInfoboxTest_007")
+  public void MercuryPortableInfoboxTest_007_HeroImageTall() {
+    PortableInfoboxObject infoboxTallImage = new PortableInfoboxObject(driver);
+    infoboxTallImage.openArticleOnWikiByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+    infoboxTallImage.isHeroImageSquare();
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_008")
```

Comment:
Remove the atomic groups annotation and add one on the top of test class definition